internal/release/postinstall.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Copyright The OpenTelemetry Authors
+# Copyright Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if command -v systemctl >/dev/null 2>&1; then
```

Comment:
I wonder if we should have an equivalent block for OpenRC, which is what Alpine uses.